src/foam/u2/crunch/FlatteningCapabilityWizardlet.js

```diff
@@ -56,6 +56,18 @@ foam.CLASS({
 
       // Return true to prevent wizardlet from being pushed normally
       return true;
+    },
+    async function save() {
+      await foam.Promise.reduce(this.delegates, d => d.save());
```

Comment:
Do you really want a 'reduce' operation? Do you really compare to combine the results? Or, would a forEach/forAll/all method be more understandable?

Reply:
Yes, a reduce is desired. I've been using the following technique whenever I expect promises to execute after each previous promise is complete:
```
newPromise = objects.reduce((p, object) =>
  p.then(() => object.someAsyncMethod()), Promise.resolve());
```

I could use `Promise.all`, but then wizardlets are saved an an arbitrary order which we don't want here - we actually want these asynchronous behaviours to occur synchronously.

The reason for adding this to `stdlib` is that I find I need this a lot, but it's not very readable when I write it as I've done in the example above.

Reply:
Reduce doesn't imply that they are executed in sequence, so maybe something like inSeq() or inSequence() or aSeq() (the same as FOAM1)?

Reply:
If I read `aSeq` I would believe it to mean "not sequential" but I like `inSeq`

Reply:
Oh, how about just `inOrder`?